community/contributing/updating_the_class_reference.rst

```diff
@@ -1,151 +1,176 @@
+Create a new branch to make your changes. It makes it a lot easier to
+synchronize your improvements with other contributors. It's also easier to clean
+up your repository if you run into any issues with Git.
 
 ::
 
     git checkout -b your-new-branch-name
 
-The new branch is the same as your master branch until you start to write API docs. In the ``doc/`` folder, you'll find the class reference.
+The new branch is the same as your master branch until you start to write API
+docs. In the ``doc/`` folder, you will find the class reference.
 
-How to keep your local clone up-to-date
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Keeping your local clone up-to-date
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Other writers contribute to Godot's documentation. Your local repository will fall behind it, and you'll have to synchronize it. Especially if other contributors update the class reference while you work on it.
+Other writers contribute to Godot's documentation. Your local repository will
+fall behind it. You will have to synchronize it, especially if other
+contributors update the class reference while working on it.
 
-First add an ``upstream`` git *remote* to work with. Remotes are links to online repositories you can download new files from.
+First, add an ``upstream`` Git *remote*. Remotes are links to online repositories
+from which you can download new files. The following command registers a new
+remote named "upstream" that links to the original Godot repository.
 
 ::
 
     git remote add upstream https://github.com/godotengine/godot
 
-You can check the list of all remote servers with:
+You can check the list of all remote servers with this command:
 
 ::
 
     git remote -v
 
-You should have two: ``origin``, your fork on GitHub that Git adds by default, and ``upstream``, that you just added:
-
-
-::
+You should have two remotes:
 
-    origin  https://github.com/your_name/godot.git (fetch)
-    origin  https://github.com/your_name/godot.git (push)
-    upstream        https://github.com/godotengine/godot.git (fetch)
-    upstream        https://github.com/godotengine/godot.git (push)
+1. ``origin``, corresponding to your fork on GitHub. Git adds it by default when
+   you clone a repository.
+2. ``upstream``, that you just added.
 
-Each time you want to sync your branch to the state of the upstream repository, enter:
+The output should look like the following:
 
 ::
 
-    git pull --rebase upstream master
-
-This command will first ``fetch``, or download the latest version of the Godot repository. Then, it will reapply your local changes on top.
+    origin https://github.com/your_name/godot.git (fetch) origin
+    https://github.com/your_name/godot.git (push) upstream
+    https://github.com/godotengine/godot.git (fetch) upstream
+    https://github.com/godotengine/godot.git (push)
```

Comment:
Messed up newlines.

I don't think this section about `git remote -v` is useful though.